### ISSUE TITLE
[Feature request] Minor: Redirect user into app upon successful registration

### ISSUE THREAD

**Issue by vktr-r2** [gbowne1/codebooker#172]
**Current Registration Flow is Very Confusing to User**

- As it currently stands, once users successfully register a new user profile they are not notified with any successful prompts, and not redirected away from the registration page
- This can cause confusion and inconvenience when registering

https://github.com/gbowne1/codebooker/assets/122321007/ee2f3c19-fa29-423f-a4e0-f05cb8734842

**Proposed Solution**
- Upon successful registration in database, login the new user and redirect them into the app

**Comment by vktr-r2** [comment]
Can you assign this one to me as well please 😊

**Comment by gbowne1** [comment]
@vktr-r2 

I believe you should get an email and probably a toast on successful registration and a Welcome @user etc.  We worked on this, but it may not be a 100% perfect process.  

I noticed we have react-toast and I think we should convert all of the react toasts, alerts and notifications to use @mui/material/snackbar instead so that we aren't overloaded with un-necessary dependencies.

**Comment by vktr-r2** [comment]
Okay I'll look to make those changes.  Thank you!

**Comment by gbowne1** [comment]
You're welcome.  I merged your PR.  In the future, Please make a short description of your changes with the PR in the first comment box of the PR, and it would be helpful if you include either screenshots or a short video with some comments about what you did.  This will be helpful for our PR reviewers.

We need to update the documents that cover this with that process, including the CONTRIBUTING.md.

**Comment by vktr-r2** [comment]
Hey @gbowne1 sorry to bother you,

I thought I submitted short descriptions/comments with both of my PRs this time.  Are they not appearing on your end?  I will look to include more screenshots/videos in the future

Also this Issue #172 ([Feature request] Minor: Redirect user into app upon successful registration) should not be closed yet,  I'm about to start work on this issue now.


[Pull 176](https://github.com/gbowne1/codebooker/pull/176) should have resolved [Issue 171](https://github.com/gbowne1/codebooker/issues/171)

[Pull 175](https://github.com/gbowne1/codebooker/pull/175) should have resolved [Issue 170](https://github.com/gbowne1/codebooker/issues/170)

**Comment by gbowne1** [comment]
Ok, I unclosed this issue.

https://github.com/gbowne1/codebooker/pull/169 did not have anything in the description:

![Screenshot 2023-09-14 at 17-46-29 Pull requests · gbowne1_codebooker](https://github.com/gbowne1/codebooker/assets/47549872/c93a63bf-6ccd-48a6-99ac-b5123cfdb44c)
